tests/python/requirements/PySafetyBearWithoutMockTest.py

```diff
@@ -0,0 +1,25 @@
+from queue import Queue
+from bears.python.requirements.PySafetyBear import PySafetyBear
+from coalib.settings.Section import Section
+from coalib.testing.LocalBearTestHelper import LocalBearTestHelper
+
+
+class PySafetyBearTest(LocalBearTestHelper):
+
+    def setUp(self):
+        self.uut = PySafetyBear(Section('name'), Queue())
+
+    def test_without_vulnerability(self):
+        self.check_validity(self.uut, ['# whee', 'foo==1.0', '# whee'])
```

Comment:
[`foo`](https://pypi.python.org/pypi/foo) doesnt have version 1.0, so that is erroneous input.

And you are testing comments here.  That should be done in a separate test method.  One test method per type of possible error.  The name of the method then says what broke.

Reply:
No not testing comment here , I make changes accordingly